package/MDAnalysis/coordinates/base.py

```diff
@@ -137,6 +140,28 @@
 from . import core
 from .. import NoDataError
 
+from ..auxiliary.base import AuxReader
+from ..auxiliary.core import get_auxreader_for
+
+class Namespace(object):
```

Comment:
I've put it in utils now. I listed it to appear in the docs in the 'Containers and lists' section since that seemed the most appropriate, but it can be moved.